pehlaschoollibrary/Classes/BookView.cpp

```diff
@@ -633,8 +633,6 @@ void BookView::popBookScene()
             //Finish the current activity and go back to Main screen
             finishActivity();
         }
-    }
```

Comment:
Is is closing brace accidentally removed?

Reply:
No the previous commit had an extra brace which happened while removing the if-else statement